e2e-tests/src/tests/bindings.test.ts

```diff
@@ -0,0 +1,638 @@
+/**
+ * Test the Action Bindings for Merlin and the Scheduler
+ *
+ * Health endpoints are already tested in health.test.ts
+ */
+import { expect, test } from '@playwright/test';
+import req, {awaitSimulation} from '../utilities/requests.js';
+import time from "../utilities/time.js";
+import * as urls from '../utilities/urls.js';
+
+test.describe('Merlin Bindings', () => {
+  let mission_model_id: number;
+  let plan_id: number;
+  let admin: User = {
+    username: "Admin_User",
+    default_role: "aerie_admin",
+    allowed_roles: ["aerie_admin"],
+    session: {'x-hasura-role': 'aerie_admin', 'x-hasura-user-id': 'Admin_User'}
+  };
+  let nonOwner: User = {
```

Comment:
Would it be useful to add a `viewer` role to check for read-only permissions on some of these endpoints?

Reply:
There's another ticket [#1053](https://github.com/NASA-AMMOS/aerie/issues/1053) to do proper permissions e2e tests on the endpoints